docs/sources/microsoft-365/outlook-cal/example-api-responses/Users_onlineMeetings_attendanceReport_v1.0.json

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
Most of the lines in the PR are related to include example files (original, sanitized, sanitized without user id, sanitized without user and group ids) for online meetings for outlook calendar tests